code/code/misc/limits.cc

```diff
@@ -467,6 +467,22 @@ int TPerson::hitGain()
   if (affectedBySpell(SPELL_ENLIVEN))
     gain *= 2;
 
+  // Scale regen based on the number of bloodlust stacks 
+  if (affectedBySpell(SKILL_BLOODLUST)) {
+    int mod = 0;
+    affectedData *ch_affected;
+
+    for (ch_affected = affected; ch_affected; ch_affected = ch_affected->next) {
+      if (ch_affected->type == SKILL_BLOODLUST) {
+        // set the mod and remove the affect so we can add it fresh
+	mod += ch_affected->modifier;
```

Comment:
bad indent

Reply:
I think I addressed the concern?  I had some tab characters in a couple indents, replaced with all space characters.  Let me know if the issue was something else.

Reply:
Yep, looks fine now.